website/docs/reference/snapshot-configs.md

```diff
@@ -222,106 +252,141 @@ snapshots:
+    <File name='dbt_project.yml'>
 
-<File name='dbt_project.yml'>
+    ```yml
 
-```yml
-snapshots:
-  jaffle_shop:
-    postgres_app:
-      orders_snapshot:
+    snapshots:
+      jaffle_shop:
         +unique_key: id
-        +strategy: timestamp
-        +updated_at: updated_at
-```
-
-</File>
-
-You can also define some common configs in a snapshot's `config` block. We don't recommend this for a snapshot's required configuration, however.
-
-<File name='dbt_project.yml'>
-
-```yml
-version: 2
-
-snapshots:
-  - name: orders_snapshot
-    config:
-      persist_docs:
-        relation: true
-        columns: true
-```
-
-</File>
+    ```
+
+    </File>
+
+  Similarly, you can use the name of an installed package to configure snapshots in that package.
+
+- #### Apply configurations to one snapshot only
+  
+  <VersionBlock lastVersion="1.8">
+  Use `config` blocks if you need to apply a configuration to one snapshot only. 
+
+    <File name='snapshots/postgres_app/orders_snapshot.sql'>
+
+    ```sql
+    {% snapshot orders_snapshot %}
+        {{
+            config(
+              unique_key='id',
+              strategy='timestamp',
+              updated_at='updated_at'
+            )
+        }}
+        -- Pro-Tip: Use sources in snapshots!
+        select * from {{ source('jaffle_shop', 'orders') }}
+    {% endsnapshot %}
+    ```
+
+    </File>
+    </VersionBlock>
+
+    <VersionBlock firstVersion="1.9">
+     <File name='snapshots/postgres_app/order_snapshot.yml'>
+
+    ```yaml
+    snapshots:
+     - name: orders_snapshot
+       relation: source('jaffle_shop', 'orders')
+       config:
+         unique_key: id
+         strategy: timestamp
+         updated_at: updated_at
+         persist_docs:
+           relation: true
+           columns: true
+    ```
+    </File>
+   Pro-tip: Use sources in snapshots: `select * from {{ source('jaffle_shop', 'orders') }}`
+    </VersionBlock>
+
+  You can also use the full resource path (including the project name, and subdirectories) to configure an individual snapshot from your `dbt_project.yml` file.
+
+  For a project named `jaffle_shop`, with a snapshot file within the `snapshots/postgres_app/` directory, where the snapshot is named `orders_snapshot` (as above), this would look like:
+
+    <File name='dbt_project.yml'>
+
+    ```yml
+    snapshots:
+      jaffle_shop:
+        postgres_app:
+          orders_snapshot:
+            +unique_key: id
+            +strategy: timestamp
+            +updated_at: updated_at
+    ```
+
+    </File>
+
+  You can also define some common configs in a snapshot's `config` block. We don't recommend this for a snapshot's required configuration, however.
+
+    <File name='dbt_project.yml'>
+
+    ```yml
+    version: 2
+
+    snapshots:
+      - name: orders_snapshot
+        config:
+          persist_docs:
+            relation: true
+            columns: true
```

Comment:
I know this was just rearranged from pre-existing documentation, but I'm not sure if it actually works!

Per docs [here](https://docs.getdbt.com/reference/resource-configs/persist_docs), I think it should be the following (but I didn't try it to confirm or deny!):
```suggestion
        +persist_docs:
          relation: true
          columns: true
```